runtimes/eden/src/lib.rs

```diff
@@ -107,7 +107,7 @@ construct_runtime! {
 		DmpQueue: cumulus_pallet_dmp_queue = 33,
 		XcmpQueue: cumulus_pallet_xcmp_queue::{Pallet, Call, Storage, Event<T>} = 34,
 		PolkadotXcm: pallet_xcm::{Pallet, Call, Storage, Event<T>, Origin, Config} = 35,
-
+		XTokens: orml_xtokens::{Pallet, Call, Storage, Event<T>} = 36,
```

Comment:
Can you please add the corresponding `list_benchmark!(list, extra, orml_xtokens, XTokens);` and `add_benchmark!(params, batches, orml_xtokens, XTokens);` as well?